src/routes/_components/status/StatusPoll.html

```diff
@@ -1,10 +1,13 @@
 <div class={computedClass} aria-busy={loading} >
   {#if voted || expired }
     <ul class="poll-choices" aria-label="{intl.pollResults}">
-      {#each options as option}
+      {#each options as option, i}
         <li class="poll-choice option">
           <div class="option-text">
-            <strong>{option.share}%</strong> <span>{@html cleanTitle(option.title)}</span>
+            <strong>{option.share}%</strong><span>{@html cleanTitle(option.title)}</span>
+            {#if own_votes == i || Object.values(own_votes).includes(i)}
+            <SvgIcon className="poll-icon" href="#fa-check" />
+             {/if}
```

Comment:
hey would you mind throwing in some text too to indicate own_votes for screen-readers?

You could lean into making it part of the design and put a visible piece of text after the mark saying something like `your vote`. or you could go a little hacky and put a 0px font-size element with text after it so it doesnt show up visible. I don't really like that because I'm not sure if there's edge cases it will cause to break; but i also don't know if there's other strats to use when you want text but don't want it part of the visual layout.

Whatever you do, I think you're not supposed to use aria-label. MDN is really insistent that aria-label is only intended for use for interactive elements, so maybe it messes up screen readers if you use them on non-interactive elements (could make it selectable as an interactive object maybe).

Reply:
Sounds fair, maybe I'll try to introduce some Title-element in the SvgIcon, that all Icons on here can be screen-reader friendly.
https://github.com/NickColley/semaphore/blob/main/src/routes/_components/SvgIcon.html#L7
It then can be indicated by an aria-describedby.
This solution seems to double the alt description in some edge cases, but seems like the best solution if, I'll stick to an SVG Element and avoid area-labels
https://www.smashingmagazine.com/2021/05/accessible-svg-patterns-comparison/#testing-results